get_all_blobs/getallblobs.js

```diff
@@ -0,0 +1,181 @@
+            "type": "bytes32"
+          },
+          {
+            "internalType": "uint256",
+            "name": "numberOfLayer1Txs",
+            "type": "uint256"
+          },
+          {
+            "internalType": "bytes32",
+            "name": "priorityOperationsHash",
+            "type": "bytes32"
+          },
+          {
+            "internalType": "bytes32",
+            "name": "bootloaderHeapInitialContentsHash",
+            "type": "bytes32"
+          },
+          {
+            "internalType": "bytes32",
+            "name": "eventsQueueStateHash",
+            "type": "bytes32"
+          },
+          {
+            "internalType": "bytes",
+            "name": "systemLogs",
+            "type": "bytes"
+          },
+          {
+            "internalType": "bytes",
+            "name": "pubdataCommitments",
+            "type": "bytes"
+          }
+        ],
+        "internalType": "struct IExecutor.CommitBatchInfo[]",
+        "name": "_newBatchesData",
+        "type": "tuple[]"
+      }
+    ],
+    "name": "commitBatchesSharedBridge",
+    "outputs": [],
+    "stateMutability": "nonpayable",
+    "type": "function"
+  }
+]
+
+const contract = new web3.eth.Contract(abi);
+
+function hexToUtf8(hex) {
+  // Remove the '0x' prefix if present
+  if (hex.startsWith('0x')) {
+    hex = hex.slice(2);
+  }
+
+  // Ensure the hex string has an even length
+  if (hex.length % 2 !== 0) {
+    throw new Error('Invalid hex string length');
+  }
+
+  // Convert hex string to a byte array
+  const bytes = [];
+  for (let i = 0; i < hex.length; i += 2) {
+    bytes.push(parseInt(hex.substr(i, 2), 16));
+  }
+
+  // Convert byte array to UTF-8 string
+  const utf8String = new TextDecoder('utf-8').decode(new Uint8Array(bytes));
+  return utf8String;
+}
+
+async function getTransactions() {
+  const latestBlock = await web3.eth.getBlockNumber();
+  const validatorTimelockAddress = "0xeacf0411de906bdd8f2576692486383797d06004";
+  const commitBatchesSharedBridge_functionSelector = "0x6edd4f12";
+  for (let i = 0; i <= latestBlock; i++) {
+    const block = await web3.eth.getBlock(i, true);
+    block.transactions.forEach(tx => {
+      if (tx.to == validatorTimelockAddress) { 
+        const input = tx.input;
+        const txSelector = input.slice(0, 10);
+        if (txSelector == commitBatchesSharedBridge_functionSelector) {
+          const functionAbi = contract.options.jsonInterface.find(item => item.name === 'commitBatchesSharedBridge');
+          const decodedParams = web3.eth.abi.decodeParameters(
+            functionAbi.inputs,
+            input.slice(10) // Remove the function selector (first 10 characters of the calldata)
+          );
+          commitment = hexToUtf8(decodedParams._newBatchesData[0].pubdataCommitments.slice(4));
+          console.log(`Decoded Commitment:`, commitment);
+        }
+      }
+    });
+  }
+}
+
+getTransactions();
+
+//0x4ed3cbf1cf6e8738118f87e5060aee0817c6f18b Chain Admin
```

Comment:
what is the purpose of these comments?

Reply:
They are all the contracts that are called on L1, I think it is nice to leave them somewhere is case we need them in the future. That could go somewhere else tough